src/main/java/com/icthh/xm/ms/balance/domain/BalanceSpec.java

```diff
@@ -14,17 +15,21 @@ public class BalanceSpec {
 
     @Data
     public static class BalanceTypeSpec {
+        private static final List<StatusSpec> EMPTY_STATUSES = emptyList();
```

Comment:
`emptyList()` itself is a constant, no need to create another variable, just return it in `requireNonNullElse` invocation

Reply:
fixed